app-services/secrets/res/configuration.toml

```diff
@@ -61,9 +62,18 @@ Timeout = "30s"
 
   [SecretStore.Authentication]
   AuthType = 'X-Vault-Token'
-  # Vault root token that's used for authentication
-  AuthToken = 's.imTIVMl1OWhlX3zMAhSyaFlO'
+  
+# Exclusive Secret Store
+[SecretStoreExclusive]
+  Host = 'localhost'
+  Port = 8200
+  Path = '/v1/secret/edgex/secretsExample'
+  Protocol = 'http'
+  Tokenfile = 'res/token.json'
```

Comment:
How is end-2-end working so far?  Hopefully, adding example-service in the docker-compose file can retrieve the token from Vault for secrets.